### PR TITLE
[Merged by Bors] - feat: one lemma about LocallyFinite

### PR DESCRIPTION
**Pull request by grunweg**
From sphere-eversion; I'm just submitting this upstream.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Topology/LocallyFinite.lean [hunk 1]
```diff
@@ -39,6 +39,14 @@ theorem point_finite (hf : LocallyFinite f) (x : X) : { b | x ∈ f b }.Finite :
   ht.subset fun _b hb => ⟨x, hb, mem_of_mem_nhds hxt⟩
 #align locally_finite.point_finite LocallyFinite.point_finite
 
+@[to_additive]
+theorem exists_finset_mulSupport_eq {M : Type*} [CommMonoid M] {ρ : ι → X → M}
+    (hρ : LocallyFinite fun i ↦ mulSupport <| ρ i) (x₀ : X) :
+    ∃ I : Finset ι, (mulSupport fun i ↦ ρ i x₀) = I := by
+  use (hρ.point_finite x₀).toFinset
+  rw [Finite.coe_toFinset]
+  exact rfl
```
Comment (eric-wieser) [inline Mathlib/Topology/LocallyFinite.lean:48]: This lemma looks out of place to me; if we really want this, it should be a lemma about `Set.Finite` that says "is equal to a coerced finset", but of course even the existence statement is silly in the presence of the explicit `toFinset`. cc @YaelDillies

Comment (YaelDillies) [reply Mathlib/Topology/LocallyFinite.lean:48]: Agreed. I think you should not have this lemma. Maybe you want `(hρ.point_finite x₀).toFinset = mulSupport (ρ · x₀)` instead, but even that looks dubious.

Comment (grunweg) [reply Mathlib/Topology/LocallyFinite.lean]: Thanks for pointing this out. I have removed this lemma.

**Comment by urkud** [comment]
Thanks! :tada:
bors merge

**Pull request closed without merging**